File: URI_1011.c
#include <stdio.h>
#include <math.h>
#define pi 3.14159
int main ()
{
    float r;
    while (scanf("%f",&r) != EOF)
    {
        printf("VOLUME = %.3f\n",(4/3.0)*pi*(r*r*r));
 
    }
    return 0;
 
}

File: URI_1011.java
import java.util.Scanner;


public class Main {

	public static void main(String[] args) {
		
		double a;
		Scanner sc = new Scanner(System.in);
		a = sc.nextDouble();
		
		System.out.printf("VOLUME = %.3f\n", (4 * 3.14159 * Math.pow(a, 3.0)) / 3);
		

	}

}
